packages/common/src/search/types/IHubSearchResult.ts

```diff
@@ -44,6 +47,16 @@ export interface IHubSearchResult extends IHubEntityBase {
    */
   geometry?: IHubGeography;
 
+  /**
+   * Raw result object returned from the search.
+   * This allows downstream processing to access
+   * additional properties that may not be
+   * explicitly defined in this interface
+   * Note: We will need to cast to the approproate type
+   * in order to access the properties
+   */
+  rawResult: IItem | IGroup | IUser | IOgcItem | IChannel;
```

Comment:
because this is not optional, this PR is a breaking change, so either add that to the commit message, or make this optional